src/silo/common/date.cpp

```diff
@@ -21,12 +23,14 @@ silo::common::Date silo::common::stringToDate(const std::string& value) {
       return 0;
    }
    auto split_position = value.find('-', 0);
-   if (split_position == value.size()) {
-      throw DateFormatException("Expect dates to be delimited by '-': " + value);
+   if (split_position == std::string::npos) {
+      SPDLOG_INFO("Expect dates to be delimited by '-': " + value + "\nIgnoring date");
```

Comment:
Shouldn't those be logged as warnings?

Reply:
I also thought about it, as for now it is not the expected case, I guess you are correct. We might change it later (when the format we officially support changes)